src/hb-subset-input.cc

```diff
@@ -448,22 +448,30 @@ hb_subset_input_pin_axis_location (hb_subset_input_t  *input,
 #endif
 #endif
 
-#ifdef HB_EXPERIMENTAL_API
 /**
  * hb_subset_preprocess
  * @input: a #hb_face_t object.
  *
  * Preprocesses the face and attaches data that will be needed by the
  * subsetter. Future subsetting operations can then use the precomputed data
- * to speed up the subsetting operation.
+ * to speed up the subsetting operation. Returns a new hb_face_t*.
  *
- * Since: EXPERIMENTAL
+ * See docs/subset-preprocessing.md for more information.
```

Comment:
This mark down file will not be linked from online documentation, if it is important I suggest converting it to DocBook and integrating it in to user manual until #3405 is sorted out.

Reply:
I think it's fine to not have it link for now.

Reply:
Or link to an online version?

Reply:
I'll do that